nexuse2e-core/src/main/java/org/nexuse2e/reporting/StatisticsCertificate.java

```diff
@@ -44,6 +46,8 @@ public StatisticsCertificate(CertificatePojo certificatePojo, boolean local) {
             this.configuredFor = partner.getName();
             this.nxPartnerId = partner.getNxPartnerId();
         }
+        this.validity = certificate.getValidity();
+        this.remainingDays = certificate.getRemainingDayCount();
```

Comment:
Should this property have the same name? `Days` vs `DayCount`

Reply:
Done.